README.md

```diff
@@ -3,4 +3,163 @@
 
 # Vumi Telegram transport
 
-A [Vumi](http://github.com/praekelt/vumi) transport for [Telegram](http://telegram.org).
+A [Vumi](http://github.com/praekelt/vumi) transport for [Telegram](http://telegram.org) and [Junebug](https://github.com/praekelt/junebug).
+
+## Helpful links
+
+- [Telegram bot API](https://core.telegram.org/bots/api)
+- [Vumi documentation](http://vumi.readthedocs.io/en/latest)
+- [Vumi GitHub repository](https://github.com/praekelt/vumi)
+- [Junebug documentation](http://junebug.readthedocs.io/en/latest)
+- [Junebug GitHub repository](https://github.com/praekelt/junebug)
+
+## Demo
+
+The following tutorial demonstrates how to get the Vumi Telegram transport up and running. We assume you are using a Unix-like operating system, as well as some very basic competency with the command line and Python. We also assume you have already created a Telegram bot and received an API token from Telegram (follow the instructions [here](https://core.telegram.org/bots) if you haven't yet done so).
+
+The transport receives inbound messages from Telegram via webhooks. However, Telegram only supports HTTPS URLs for webhook setup. In production, you would need to run the transport behind a load balancer or proxy server - in this tutorial, we will be using [ngrok](https://ngrok.com).
+
+Download it from the [ngrok website](https://ngrok.com/download) and follow the installation instructions. To test that the install was successful, run the following command:
+
+```
+$ ngrok http 8000
+```
+
+You should see the ngrok command line interface displayed.
+
+Now it's time to install the transport. First, navigate to your working directory. You should probably set up a virtual environment at this point - see the [virtualenv documentation](https://virtualenv.pypa.io/en/stable/) if you're not sure how. This isn't strictly necessary, but the rest of the tutorial will assume you're working inside a virtual environment. Ignore the `(ve)` prefixes if you aren't, or if you've named yours differently.
+
+Once your virtual environment is set up, install the transport using pip with the following command:
+
+```
+(ve) $ pip install -e git+git://github.com/praekelt/vumi-telegram.git@develop#egg=vumi-telegram
```

Comment:
Would be good to get this to just be a `pip install vxtelegram`. Once we have the `setup.py`, I can do a release.